src/html/login/login.php

```diff
@@ -1,15 +1,18 @@
 <?php
 require('/var/www/app/config/db_connect.php');
+session_start();
 
 $email = htmlspecialchars($_POST['email']);
 $password = htmlspecialchars($_POST['password']);
 
 if ($email and $password) {
-  $stmt = $db->prepare('SELECT COUNT(*) count FROM users WHERE email = :email AND password = :password');
+  $stmt = $db->prepare('SELECT * FROM users WHERE email = :email AND password = :password');
   $stmt->execute([':email' => $email, ':password' => $password]);
   $result = $stmt->fetch(PDO::FETCH_ASSOC);
   // var_dump($result);
-  if ($result['count'] !== '0') {
+  if ($result) {
+    $_SESSION['isloginToShortBbs'] = true;
+    $_SESSION['username'] = $result['name'];
     header('Location: /home');
     exit;
```

Comment:
```suggestion
    exit();
```
これも正しい書き方だけど、カッコつきで統一したいかも